From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 17 Aug 2023 01:53:28 +0200
Subject: ci: Don't disable modular repos in toolbox image

They are no longer included in F39, so trying to disable them
fails.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2897>
(cherry picked from commit cd1f8ba11d845b166198402540381660dfb3f083)
---
 .gitlab-ci/build-toolbox-image.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitlab-ci/build-toolbox-image.sh b/.gitlab-ci/build-toolbox-image.sh
index 8e6b853..b2d3a1f 100755
--- a/.gitlab-ci/build-toolbox-image.sh
+++ b/.gitlab-ci/build-toolbox-image.sh
@@ -35,7 +35,7 @@ build_container() {
     abattis-cantarell-fonts # system font
     gnome-backgrounds # no blank background!
   )
-  buildah run $build_cntr dnf config-manager --set-disabled '*-modular,*-openh264'
+  buildah run $build_cntr dnf config-manager --set-disabled '*-openh264'
   buildah run $build_cntr dnf install -y "${extra_packages[@]}"
   buildah run $build_cntr dnf clean all
   buildah run $build_cntr rm -rf /var/lib/cache/dnf
